examples/network/edgeStyles/selfReference.html

```diff
@@ -27,23 +27,99 @@
                 id: 1,
                 label:
                     "I'm an alligator,\nI'm a mama-papa coming for you.\nI'm the space invader,\nI'll be a rock 'n' rollin' bitch for you",
-                x:0,
-                y:0
+                x: 0,
+                y: 400
             },
             {
                 id: 2,
-                label:
-                    "Test shape",
-                widthConstraint: { minimum: 100 }, heightConstraint: { minimum: 100 },
-                x:200,
-                y:200
+                label: "Test shape",
+                widthConstraint: { minimum: 100 },
+                heightConstraint: { minimum: 100 },
+                x: 400,
+                y: 400
+            },
+            {
+                id: 3,
+                label: "This is another test shape.",
```

Comment:
It would be great to use the labels to describe what configuration was used to achieve this effect.